cdap-app-fabric/src/main/java/io/cdap/cdap/security/TokenSecureStoreRenewer.java

```diff
@@ -71,8 +69,7 @@ public class TokenSecureStoreRenewer extends SecureStoreRenewer {
     this.cConf = cConf;
     this.locationFactory = locationFactory;
     this.secureStore = secureStore;
-    this.secureExplore = cConf.getBoolean(Constants.Explore.EXPLORE_ENABLED)
```

Comment:
`secureExplore` evaluated to `false` as  `EXPLORE_ENABLED` config was `false`.
This makes `secureToken` meaningless and can be removed.